frontend/src/lib/api/index.js

```diff
@@ -0,0 +1,45 @@
+import qs from 'querystring';
+import { fetchX } from 'lib/http';
+
+const API_BASE = 'https://bkach.com/api';
```

Comment:
Could we make this an environment variable?

Conscious that the server might not want to use the fully qualified name, especially if i'm trying to run a local instance.

Not a blocker tho, might be something for me to explore.

Reply:
We should, I was just being lazy